Gopkg.toml

```diff
@@ -140,7 +140,7 @@ ignored = ["github.com/uber/cadence/.gen"]
   version = "1.1.0"
 
 [[constraint]]
-  name = "github.com/hashicorp/go-version"
+  name = "github.com/hashicorp/go-version.git"
```

Comment:
Was this intentional? I think Wenquan just removed this.

Reply:
Will remove this after I merge master branch into my branch. My base commit doesn't have this fix https://github.com/uber/cadence/pull/2319.